src/components/Resources/Card.tsx

```diff
@@ -0,0 +1,126 @@
+import { FunctionComponent } from 'react'
+
+import classNames from 'classnames'
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import StarIcon from 'mdi-react/StarIcon'
+import Link from 'next/link'
+
+import { buttonStyle, buttonLocation } from '@data'
+
+import { Resource } from '.'
+
+interface Card {
+    resource: Resource
+}
+
+interface BackgroundVariant {
+    [key: string]: string
+}
+
+interface FilterPill {
+    text: string
+    light?: boolean
+}
+
+const FilterPill: FunctionComponent<FilterPill> = ({ text, light = false }) => {
+    const background = light ? 'tw-bg-gray-100' : 'tw-bg-gray-200'
+    return (
+        <div
+            className={`${background} tw-text-gray-500 tw-py-1 tw-px-2 tw-rounded-md tw-mr-2 tw-mb-2 tw-whitespace-nowrap first-letter:tw-capitalize tw-font-mono tw-text-xs`}
+        >
+            {text}
+        </div>
+    )
+}
+
+/**
+ * This is the Card component rendered as a resource on the resources page.
+ *
+ * @param props - component props
+ * @param props.resource - a resource data object defined in resourceItems
+ */
+export const Card: FunctionComponent<Card> = ({ resource }) => {
+    // CTA text mapping
+    const watchNow = new Set(['virtual event', 'video'])
+    const readThe = new Set(['blog post', 'guide', 'customer story'])
+    let ctaText = 'Read more'
+    if (watchNow.has(resource.contentType)) {
+        ctaText = 'Watch now'
+    } else if (readThe.has(resource.contentType)) {
+        ctaText = `Read the ${resource.contentType === 'customer story' ? 'story' : resource.contentType}`
+    }
+
+    // Content Type background mapping
+    const backgrounds: BackgroundVariant = {
+        'virtual event': 'tw-from-vermillion-100 tw-to-violet-400',
+        video: 'tw-from-violet-100 tw-to-cerise-300',
+        'blog post': 'tw-from-violet-200 tw-to-vermillion-300',
+        guide: 'tw-from-blue-200 tw-to-violet-400',
+        'customer story': 'tw-from-green-100 tw-to-blue-300',
+    }
+
+    return (
+        <div className="tw-bg-white tw-shadow-md tw-rounded-lg md:tw-min-h-[469px]">
+            <div
+                className={classNames(
+                    'tw-h-[10%] tw-overflow-hidden tw-rounded-t-lg tw-flex tw-items-center tw-px-sm tw-bg-gradient-to-r',
+                    backgrounds[resource.contentType]
```

Comment:
I think we should add in a default bg fallback, even though it's not likely to happen. The strings value is so open, a typo can cause this value to be undefined. If we use an object for the values that'd ensure some more reliability. What do you think?

Reply:
Yeah that sounds good! It's probably a good idea in case we forget to update the object dictionary mapping for newly added contentTypes in our union type.